go/cmd/gno_social_feed_indexer/main.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/TERITORI/teritori-dapp/go/cmd/gno_social_feed_indexer/clientql"
+	"github.com/TERITORI/teritori-dapp/go/internal/indexerdb"
+	"github.com/TERITORI/teritori-dapp/go/pkg/networks"
+	"github.com/go-co-op/gocron"
+	"github.com/peterbourgon/ff/v3"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+)
+
+func main() {
+	// handle args
+	fs := flag.NewFlagSet("teritori-indexer", flag.ContinueOnError)
+	var (
+		dbHost       = fs.String("db-indexer-host", "", "host postgreSQL database")
+		dbPort       = fs.String("db-indexer-port", "", "port for postgreSQL database")
+		dbPass       = fs.String("postgres-password", "", "password for postgreSQL database")
+		dbName       = fs.String("database-name", "", "database name for postgreSQL")
+		dbUser       = fs.String("postgres-user", "", "username for postgreSQL")
+		networksFile = fs.String("networks-file", "networks.json", "path to networks config file")
+		networkID    = fs.String("gno-network-id", "devLocal", "network id to index")
+		txIndexerURL = fs.String("gno-tx-indexer-endpoint", "http://localhost:8546/graphql/query", "Tx indexer GraphQL endpoint")
```

Comment:
you should get this from the network definition

Reply:
network definition is a cross-language thing (we should probably transition to protobuf for this at some point btw)

I added the field in typescript and defined it for the gnodev network

changes are in https://github.com/TERITORI/teritori-dapp/pull/1246/commits/a43d98f0295bbe05ee365d9cdd092748c317f1b2 if you're curious